### ISSUE TITLE
Support handwritten notes on tablet

### ISSUE THREAD

**Issue by charlotte-chiang** [anytypeio/roadmap#24]
**High level:**
Support more devices & functionality for handwritten content

**Related Threads/Feature requests**
https://community.anytype.io/t/support-for-handwritten-notes/1030

**Comment by SEELE0** [comment]
I'm looking forward to this feature, thanks to the developer 🥳
Also I wonder to know weather hand written is similar to this  [project](https://github.com/toeverything/AFFiNE)
Look like this
![](https://affine.pro/_next/static/media/shape.6bc90d7f-opt-1080.WEBP)

**Comment by charlotte-chiang** [comment]
@SEELE0 Thanks for your thoughts! We do not know exactly how it will look yet; our team needs to do some research on expectations & needs first, to understand how we implement.

**Comment by SEELE0** [comment]
@charlotte-chiang 
Thanks for your reply, looking forward to the final effect of the new function🎉🤩

**Comment by aeluabik** [comment]
That's my dream feature ☺️

**Comment by Cossackx** [comment]
Take a look at the Excalidraw + Excalibrain plug-in for Obsidian, @charlotte-chiang .

**Comment by Cossackx** [comment]
Hoping this gets prioritized sooner.

**Comment by shuminque** [comment]
这确实是一个好的想法，具体实现是要怎么实现？

**Comment by elementh** [comment]
I'm looking forward to this, but also having the same interface on tablets (iPads & Tabs, for example) as we have on desktop, I feel like the current tablet interface is very limiting.

**Comment by sudo-saif** [comment]
Whenever you plan to work on this feature, please make it available for Windows as well, I have a graphic tablet connected to my PC and I would love taking hand notes on my PC :)

**Comment by god-boat** [comment]
me too, surface pro 8 and s23 ultra :) sad it got moved to backlog :(

**Comment by mrwildenfree** [comment]
Would also appreciate this feature — I actually mentioned it in the feedback form earlier today before I realized it was already here on the roadmap.

**Comment by jrcsb** [comment]
Very much looking forward to this feature. A game changer would be enabling functionality to convert handwriting blocks to text - Nebo is a great app for doing this - if that functionality could be replicated it would be brilliant.

**Comment by mrwildenfree** [comment]
> Very much looking forward to this feature. A game changer would be enabling functionality to convert handwriting blocks to text - Nebo is a great app for doing this - if that functionality could be replicated it would be brilliant.

It’s actually already possible on iPadOS to write with an Apple Pencil & have those scribbles turn into typed text — Apple made that a native feature & it actually works in more places than people might assume.

**Comment by Tumypmyp** [comment]
I used Logseq's whiteboards based on tldraw. It is open source and I like the look of vector notes and drawings.

**Comment by AlexCuan** [comment]
If this feature is performed, I'll be all in into Anytipe for sure!

**Comment by Fefuchs** [comment]
If this is remotely close to the features of Onenote, then I'll be switching to Anytype in an instant.

**Comment by Yu-jinKim** [comment]
Even having basic blocks for handwritten notes would be awesome. Referring back to sketches within Anytype would be the best

**Comment by realmenwearpink2** [comment]
This is the only feature keeping me from switching to Anytype!
Sad to see this was opened nearly a year ago and is still on the backlog :(

**Comment by eleazar7** [comment]
I hope this is also apart of the “Canvas/whitebroad” feature. It could really all be as one you know.

**Comment by EnvironmentalBit3940** [comment]
Really like how it's implemented in [affine](https://github.com/toeverything/AFFiNE), only thing what stops me to completely move to anytype

**Comment by 8flux** [comment]
this is the only feature missing in anynote. till then i can't fully switch

**Comment by Jawisr** [comment]
Would be great to take notes directly over the blocks as we can do in OneNote. Or having a canvas like excalidraw. Please add support for Windows Surface Pen!!

**Comment by BurntXela** [comment]
Maybe have 2 files associated with the note? Have the actual text on the bottom & the handwritten notes overlayed on top of them?

**Comment by slimhk45** [comment]
Compatibility with Obsidian's JSONCanva would be much appreciated. 

https://jsoncanvas.org/

**Comment by slimhk45** [comment]
> I hope this is also apart of the “Canvas/whitebroad” feature. It could really all be as one you know.

+1

**Comment by Perlsowisdom** [comment]
This seems a better solution than Obsidian's canvas standard ([OCWG](https://github.com/ocwg)), since its devs apparently have no desire to further extend its functionality.

**Comment by xchubbsx** [comment]
Is there any update on priority for this feature to be added? Are there any current workouts with external integrations?

**Comment by BIGBEASTISHANK** [comment]
You can integrate excalidraw with it, we need a board

**Comment by eleazar7** [comment]
yes, this would be a huge boost to the community once they drop this feature.

**Comment by dotsupershow** [comment]
I would really love some solid support for handwritten notes and pressure sensitivity!

**Comment by trymeouteh** [comment]
This feature would be a nice addition and with AI integration, a feature could be added using AI to convert the handwritten note into a text note

**Comment by EchoBlr** [comment]
I don't use Anytype and no canvas/whiteboard type is what stopping me. I'm coming to check on any changes in roadmap once in a few month and still there's nothing, which is sad. 

Really looking into some changes in the future

**Comment by 8flux** [comment]
Seems like a half dead project to me considering handwritten notes was already requested 2 years ago which is a deal breaker for most of us.

**Comment by pirasalbe** [comment]
This might be related: https://github.com/anyproto/anytype-ts/issues/548

**Comment by Leninhoo** [comment]
We'd love to see this, isn't it literally in the plans?

**Comment by Fefuchs** [comment]
Looks like this Feature isn't on the roadmap anymore!?

**Comment by ag0n1k** [comment]
@Fefuchs They’ve reworked the roadmap (at least that’s my understanding). This task is on their radar, but it’s being treated as an experimental rollout—Android tablets first, then the rest. Expect ~1–2 years. More info: https://www.youtube.com/watch?v=3_7OcgHY49Y
